shell/common/shell.h

```diff
@@ -423,7 +483,8 @@ class Shell final : public PlatformView::Delegate,
 
   Shell(DartVMRef vm, TaskRunners task_runners, Settings settings);
 
-  static std::unique_ptr<Shell> CreateShellOnPlatformThread(
+  static void CreateShellOnPlatformThread(
```

Comment:
Document this function especially on how the new `shell_holder_promise` parameter should be given and used later.